pngToNp.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
I feel you don't need a separate note book for this conversion.
Why don't you add this piece of code at the end of  GLNetSetup.ipynb ?